app/src/lib/stores/app-store.ts

```diff
@@ -5105,16 +5111,23 @@ export class AppStore extends TypedBaseStore<IAppState> {
     repository: Repository,
     branch: Branch
   ): Promise<void> {
-    const gitHubRepository = repository.gitHubRepository
+    const { gitHubRepository } = repository
+
     if (!gitHubRepository) {
       return
     }
 
-    const urlEncodedBranchName = escape(branch.nameWithoutRemote)
-    const baseURL = `${
-      gitHubRepository.htmlURL
-    }/pull/new/${urlEncodedBranchName}`
+    const { parent: parentGitHubRepository } = gitHubRepository
 
+    const urlEncodedBranchName = escape(branch.nameWithoutRemote)
+    const urlEncodedOwner = escape(gitHubRepository.owner.login)
+
+    const baseURL =
+      gitHubRepository.fork && parentGitHubRepository
+        ? `${
+            gitHubRepository.htmlURL
+          }/compare/${urlEncodedOwner}:${urlEncodedBranchName}`
```

Comment:
Could you elaborate on why we're using the `/compare` endpoint here for forks vs the old `/pull/new`?